INSTALL.md

```diff
@@ -0,0 +1,10 @@
+# Install Zync
+
+Download this repository and run `bundle install`.
```

Comment:
maybe offer this as 2 commands that people can easily copy paste?
```
git clone ...
bundle install
```

Reply:
This should not be needed at all. `bin/setup` should handle the project setup.